.github/workflows/release.yml

```diff
@@ -559,13 +566,77 @@ jobs:
           path: installer/*.msi
           if-no-files-found: error
 
+#
+# Windows Cryptomator.exe bundle
+#
+  win-exe:
+    name: Build Cryptomator.exe bundle
+    runs-on: windows-latest
+    needs: [win-msi, metadata]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download Windows msi
+        uses: actions/download-artifact@v2
+        with:
+          name: win-msi
+          path: dist/win/bundle/resources
+      - name: Strip version info from msi file name
+        run: mv dist/win/bundle/resources/Cryptomator*.msi dist/win/bundle/resources/Cryptomator.msi
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: ${{ env.JAVA_VERSION }}
+          cache: 'maven'
+      - name: Generate license
+        run: >
+          mvn -B license:add-third-party
+          "-Dlicense.thirdPartyFilename=license.rtf"
+          "-Dlicense.fileTemplate=dist/win/bundle/resources/licenseTemplate.ftl"
+          "-Dlicense.outputDirectory=dist/win/bundle/resources"
```

Comment:
redundant to dependent job `win-msi`

Reply:
No, it is not. The msi license is not the same as the one included of the bundle. The bundle license contains a note about WinFsp, while the MSI license is just an rtf copy of the `THIRD-PARTY.txt`.